templates/redis/redis-statefulset.yaml

```diff
@@ -36,6 +36,8 @@ spec:
         cluster-autoscaler.kubernetes.io/safe-to-evict: "true"
         {{- end }}
     spec:
+      securityContext:
+        RunAsUser: 999
```

Comment:
This should be `runAsUser` right?

Please add tests too

Reply:
Also, while 999 is right for the official image, we may want to expose it as a parameter in case someone wants to use another redis image.

That said, the official image uses gosu to drop root anyways 🤷‍♂️